broker-core/src/main/java/org/apache/qpid/server/util/FileUtils.java

```diff
@@ -205,8 +205,10 @@ public static void copy(File src, File dst)
      */
     public static void copyCheckedEx(File src, File dst) throws IOException
     {
-        InputStream in = new FileInputStream(src);
-        copy(in, dst);
+        try (InputStream in = new FileInputStream(src))
```

Comment:
The `InputStream` is closed in `copy`. Arguably the method `copy` should indicate that it closes the input stream. It make more sense to rename it into `copyInputStreamIntoFileAndClose`. Though, I suppose for a better code relaibility the 'try-with-resource` would be good here.